openfold/config.py

```diff
@@ -152,9 +152,42 @@ def model_config(
         c.model.template.enabled = False
         c.model.heads.tm.enabled = True
         c.loss.tm.weight = 0.1
+    # SINGLE SEQUENCE EMBEDDING PRESETS
+    elif name == "seqemb_initial_training":
+        c.data.train.max_msa_clusters = 1
+        c.data.eval.max_msa_clusters = 1
+        c.data.train.max_distillation_msa_clusters = 1
+    elif name == "seqemb_finetuning":
+        c.data.train.max_msa_clusters = 1
+        c.data.eval.max_msa_clusters = 1
+        c.data.train.max_distillation_msa_clusters = 1
+        c.data.train.crop_size = 384
+        c.loss.violation.weight = 1.
+        c.loss.experimentally_resolved.weight = 0.01
+    elif name == "seq_model_esm1b":
+        c.data.common.use_templates = True
+        c.data.common.use_template_torsion_angles = True
+        c.model.template.enabled = True
+        c.data.predict.max_msa_clusters = 1
+    elif name == "seq_model_esm1b_ptm":
+        c.data.common.use_templates = True
+        c.data.common.use_template_torsion_angles = True
+        c.model.template.enabled = True
+        c.data.predict.max_msa_clusters = 1
+        c.model.heads.tm.enabled = True
+        c.loss.tm.weight = 0.1
     else:
         raise ValueError("Invalid model name")
 
+    if name.startswith("seq"):
```

Comment:
Could we change this to be `name.startswith("seqemb")`, and change the two other presets `seq_model_esm1b` and `seq_model_esm1b_ptm` to begin with seq_emb?

This will help in case future presets unrelated to solo seq also begin with 'seq'